misc/python/materialize/parallel_workload/parallel_workload.py

```diff
@@ -0,0 +1,139 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import argparse
+import datetime
+import os
+import random
+import sys
+import threading
+import time
+from typing import Optional
+
+import pg8000
+
+from materialize.parallel_workload.action import ddl_actions, dml_actions
+from materialize.parallel_workload.database import Database
+from materialize.parallel_workload.worker import Worker
+
+SEED_RANGE = 1_000_000
```

Comment:
I am not sure what the benefit to this constant is. Just generate a seed over the entire int range.

Reply:
There is no limit to int range in Python, so I have to choose one.